backend/danswer/server/query_and_chat/query_backend.py

```diff
@@ -136,6 +142,109 @@ def query_validation(
     return QueryValidationResponse(reasoning=reasoning, answerable=answerable)
 
 
+@basic_router.get("/user-searches")
+def get_user_query_sessions(
```

Comment:
would standardize naming wherever possible, like is query and search the same thing? I assume they are

Reply:
Great point- don't want to create another assistants vs persona!